README.md

```diff
@@ -54,7 +67,7 @@ The software uses PyTorch + NumPy FFT on CPU, and PyTorch + CuPy + CuFFT on GPU.
 
 If you use this code in your work please cite our paper:
 
-The scattering authors, [*Kymatio: Fast Scattering in 1-D,2-D,3-D*]()
+~~ The scattering authors, [*Kymatio: Fast Scattering in 1-D,2-D,3-D*]() ~~
```

Comment:
The paper is not out yet, so I would be in favor of removing this line altogether

Reply:
Alright! Feel free to PR